lib/modules/manager/composer/readme.md

```diff
@@ -1,3 +1,10 @@
+<!-- prettier-ignore -->
```

Comment:
It looks like we implemented a workaround for Composer `2.5.0` with these PRs:

- #19518
- #19527

Do we still need the warning admonition?